specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/realm.auth.cred.required.username.no.password/response.rpt

```diff
@@ -0,0 +1,10 @@
+accept http://localhost:8000/resource
+accepted
+connected
+
+read method "GET"
+read header "Authorization" "Basic am9lOmJhZHBhc3M=" #login joe:badpass 

```

Comment:
if this bad password or no password? Need to reconcile the comment and the script file name